ramips: remove useless resets properties from SoC dtsi

DragonBluep opened the pull request:
These drivers don't request reset control. And most reset properties even have incorrect reset source definitions.

1. interrupt controller, ref: arch/mips/ralink/irq.c
2. memory controller, ref: arch/mips/ralink/of.c
3. gpio controller, ref: drivers/gpio/gpio-ralink.c (local patch)
4. systic, ref: arch/mips/ralink/cevt-rt3352.c
5. pwm, ref: drivers/pwm/pwm-mediatek-ramips.c (local patch)

Tested on MT7620 && MT7628.

DragonBluep committed:
ramips: remove useless resets properties from SoC dtsi

These drivers don't request reset control. And most reset properties
even have incorrect reset source definitions.

1. interrupt controller, ref: arch/mips/ralink/irq.c
2. memory controller, ref: arch/mips/ralink/of.c
3. gpio controller, ref: drivers/gpio/gpio-ralink.c (local patch)
4. systic, ref: arch/mips/ralink/cevt-rt3352.c
5. pwm, ref: drivers/pwm/pwm-mediatek-ramips.c (local patch)

Tested on MT7620 && MT7628.

Signed-off-by: Shiji Yang <yangshiji66@qq.com>

neheb commented:
hrm this node is not present upstream in mt7621.dtsi...

DragonBluep commented:
> hrm this node is not present upstream in mt7621.dtsi...

You mean `systick`? Yes, it does nothing here. MT7621 doesn't use `"ralink,cevt-systick"` driver at all. And there are also some nodes such as `pcm` seem to be useless.

paraka commented on target/linux/ramips/dts/mt7621.dtsi:
```diff
@@ -124,9 +124,6 @@
 			compatible = "ralink,mt7621-systick", "ralink,cevt-systick";
```
You can remove the whole node if you want. There is no sense in having this for mt7621 based boards.

DragonBluep replied:
Thank you, I have removed it.

DragonBluep committed:
ramips: remove systick node for mt7621 SoC dtsi

This node is useless because MT7621 uses the generic mips systick
driver instead of the ralink systick driver.

Signed-off-by: Shiji Yang <yangshiji66@qq.com>
